quora-service/src/main/java/com/upgrad/quora/service/entity/User.java

```diff
@@ -0,0 +1,184 @@
+package com.upgrad.quora.service.entity;
+
+import org.apache.commons.lang3.builder.ToStringExclude;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "users")
+@NamedQueries({
+  @NamedQuery(name = "userByEmail", query = "select u from User u where u.email =:email"),
+  @NamedQuery(name = "userByUsername", query = "select u from User u where u.userName =:username"),
+  @NamedQuery(name = "userByUuid", query = "select u from User u where u.uuid = :uuid")
+})
+public class User implements Serializable {
```

Comment:
Can we add Entity as  suffix for all entity classes....as it will be inline with our suggested tutorial

Reply:
I did not add because while making an instance of a class, we tend to give it the same name as class and so if we name it UserEntity, it would be UserEntity userEntity = new UserEntity(); 
While entity typically represents a table, userEntity is an intense. I think instead of userEntity, it can be better called user or userEnityInstance.
Let me know your views on this!